lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1908,6 +1908,10 @@ defmodule Kernel.SpecialForms do
 
     * `0` - if there is no matching message in the mailbox, the timeout
       will occur immediately
+      
+  The timeout value must be a positive integer smaller than 0xffffffff or
```

Comment:
Maybe the following will read better:
```
smaller than `4_294_967_295` (`0xFFFFFFFF` in hex notation)
```
_Note, I used uppercase in hex notation: https://github.com/lexmag/elixir-style-guide#hex-literals._